src/issuer-helper.ts

```diff
@@ -46,6 +46,7 @@ const credentialRequestHandler = async (issuerId: string, holderId: string, acce
     CREDENTIAL_NAME: credentialRecord.name,
     CREDENTIAL_DESC: credentialRecord.description,
     ISSUANCE_DATE: credentialRecord.issuanceDate,
+    ...(credentialRecord.expirationDate ? { EXPIRATION_DATE: credentialRecord.expirationDate } : null),
```

Comment:
@dmitrizagidulin This spread syntax includes the expiration date only if it is defined on the credential. Let me know how you feel about this (e.g., should I just assume that it will the field will always be defined like the others, do you think I should add this logic to the other fields, should I split this into a function).

Reply:
Hmm, I'm not sure how the `...null` part would work (if there was no expiration date).
Yeah, let's just set the expiration date in a function. (I will not always be defined.)

Reply:
Yeah, I coincidentally had a similar problem in another codebase recently and came across the `...null` solution after doing a bit of digging around the web. I also tested it out in my terminal locally and I'm confident this works as expected. As for the function, my plan was to put everything in the parenthesis into a function, so that this line would read something like `...getExpirationDate(credentialRecord)`. Does that make sense?

Reply:
While I was right about `...null` conditionally including expiration date from `credentialConfig`, I also needed to add the following to the template file (and consequently convert it from a `.json` file to a `.txt` file) if we don't want to include an empty string for the expiration date in the credential (which breaks schema validation for the credential):
```
{{#if EXPIRATION_DATE}}
  "expirationDate": "{{EXPIRATION_DATE}}",
{{/if}}
```

Reply:
I think I'd like to see something simpler. So, in the template, don't have an expiration date in there at all.
And then in the processing code, something like:

```js
if (credentialRecord.expirationDate) {
  vc.expirationDate = credentialRecord.expirationDate
}
```